stdlib/docker/src/main/ballerina/src/docker/annotation.bal

```diff
@@ -16,22 +16,22 @@
 
 # Docker annotation configuration.
 #
-# + registry - Docker registry url.
-# + name - Name of the docker image. Default value is the file name of the executable .jar file.
+# + registry - Docker registry URL.
+# + name - Name of the Docker image. Default value is the file name of the executable .jar file.
 # + tag - Docker image tag. Default value is `"latest"`.
-# + username - Username for docker registry.
-# + password - Password for docker registry.
-# + baseImage - Base image to create the docker image. Default value is `"openjdk:8-jre-alpine"`.
-# + buildImage - Enable building docker image. Default value is `true`.
-# + push - Enable pushing docker image to registry. Field `buildImage` must be set to `true` to be effective. Default value is `false`.
-# + cmd - Value for CMD for the generated Dockerfile. Default is `CMD java -jar ${APP} [--b7a.config.file=${CONFIG_FILE}] [--debug]`.
-# + enableDebug - Enable ballerina debug. Default is `false`.
-# + debugPort - Ballerina remote debug port. Default is `5005`.
+# + username - Username for the Docker registry.
+# + password - Password for the Docker registry.
+# + baseImage - Base image to create the Docker image. Default value is `"openjdk:8-jre-alpine"`.
+# + buildImage - Enable building the Docker image. Default value is `true`.
+# + push - Enable pushing the Docker image to registry. The value of the `buildImage` field must be set to `true` to be effective. Default value is `false`.
+# + cmd - CMD value for the generated Docker file. Default is `CMD java -jar ${APP} [--b7a.config.file=${CONFIG_FILE}] [--debug]`.
+# + enableDebug - Enable Ballerina debugging. Default is `false`.
+# + debugPort - Ballerina remote debugging port. Default is `5005`.
 # + dockerAPIVersion - Docker API version.
-# + dockerHost - Docker host IP and docker PORT. (e.g minikube IP and docker PORT).
-# Default is to use DOCKER_HOST environment variable.
-# If DOCKER_HOST is unavailable, use `"unix:///var/run/docker.sock"` for Unix or use `"npipe:////./pipe/docker_engine"` for Windows 10 or use `"localhost:2375"`.
-# + dockerCertPath - Docker certificate path. Default is to use `"DOCKER_CERT_PATH"` environment variable.
+# + dockerHost - Docker host IP and Docker port (e.g,. Minikube IP and Docker port).
+# Default is to use the DOCKER_HOST environment variable.
```

Comment:
```suggestion
# Default is to use the `DOCKER_HOST` environment variable.
```

Reply:
Default behavior is to use the `DOCKER_HOST` environment variable

Reply:
> Default behavior is to use the `DOCKER_HOST` environment variable

Added.